lib/utils/encoding/sdkSuffix/getSDKSuffix.js

```diff
@@ -0,0 +1,52 @@
+let fs = require('fs');
+let path = require('path');
+let encodeVersion = require('./encodeVersion');
+
+let nodeSDKValue = 'M'; // Constant per SDK
```

Comment:
`nodeSDKValue` ?  maybe let's give it a more generic name, so the code would be much similar with other SDKs.

Like `SDKCode` or `SDKType` or what is written in spec.